src/execution_plan/ops/op_expand_into.c

```diff
@@ -58,6 +58,7 @@ static void _populate_filter_matrix(OpExpandInto *op) {
 		/* Update filter matrix F, set row i at position srcId
 		 * F[i, srcId] = true. */
 		Node *n = Record_GetNode(r, op->srcNodeIdx);
+		assert(n && "failed to resolve source node for ExpandInto");
```

Comment:
You're guarantee `n` will exists, see `if(!Record_GetNode(childRecord, op->srcNodeIdx)) {` `consume` function.